src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -272,13 +274,42 @@
                     Width="Auto"
                     MinWidth="40"
                     x:Load="{x:Bind ViewModel.IsImage, Mode=OneWay, FallbackValue=False}"
-                    Command="{x:Bind ViewModel.SetAsBackgroundCommand, Mode=OneWay}"
-                    Label="{x:Bind ViewModel.SetAsText, Mode=OneWay}"
+                    Label="{helpers:ResourceString Name=SetAsBackgroundFlyout}"
                     LabelPosition="Default"
-                    ToolTipService.ToolTip="{helpers:ResourceString Name=SetAsBackground}">
+                    Style="{StaticResource ToolBarAppBarButtonFlyoutStyle}"
+                    ToolTipService.ToolTip="{helpers:ResourceString Name=SetAsBackgroundFlyout}">
                     <AppBarButton.Icon>
                         <FontIcon Glyph="&#xE91B;" />
                     </AppBarButton.Icon>
+                    <AppBarButton.Flyout>
+                        <MenuFlyout Placement="Bottom">
+                            <MenuFlyoutItem
+                                x:Name="SetAsBackgroundFlyoutItem"
+                                Command="{x:Bind ViewModel.SetAsBackgroundCommand, Mode=OneWay}"
+                                Text="{helpers:ResourceString Name=SetAsBackground}">
+                                <MenuFlyoutItem.Icon>
+                                    <FontIcon Glyph="&#xE91B;" />
+                                </MenuFlyoutItem.Icon>
+                            </MenuFlyoutItem>
+                            <MenuFlyoutItem
+                                x:Name="SetAsSlideshowFlyoutItem"
```

Comment:
Looking really good! Do you think this should be a dedicated toolbar button so that users don't have to open the flyout to access a single option?

Reply:
I like that idea If I'm understanding correctly. When you select one image it shows the Flyout for lockscreen and background, but when you select multiple it hides the flyout and replaces with single Slideshow button.

Reply:
That's exactly what I'm imagining 👀

Reply:
I like to call this the intuitive toolbar

Reply:
I'll make those changes.